deploy/helm/sumologic/templates/_helpers.tpl

```diff
@@ -51,20 +51,20 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
 {{- end -}}
 {{- end -}}
 
-{{- define "sumologic.logs.collector.name" -}}
-{{- template "sumologic.fullname" . }}-otelcol-logs-collector
+{{- define "sumologic.metadata.name.logs.collector.configmap" -}}
```

Comment:
I was under the impression that the `metadata` namespace was for metadata enrichment services. otellogs don't really fit in there.

Reply:
it can be my mistake, I was for me k8s `metadata.name` :sweat_smile:

Reply:
https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/54b7e32b79cf629b5977ce075516a6333dffbece/deploy/helm/sumologic/templates/_helpers.tpl#L370-L388

Reply:
so it is about kubernetes
```
metadata:
  name:
```